apps/web/.linguirc

```diff
@@ -23,7 +23,8 @@
     "en",
     "es",
     "ta",
-    "zh"
+    "zh",
+    "kn"
```

Comment:
may be you should have dont `in-kn`?  now how will you give code for korean? 😅

Reply:
<img width="488" alt="Screenshot 2023-04-09 at 1 15 10 PM" src="https://user-images.githubusercontent.com/69431456/230760688-810fa114-6059-4caa-99d2-6efe22f2c7b3.png">

It's a language standard code

Reply:
🤔